netlify.toml

```diff
@@ -0,0 +1,18 @@
+[build]
+base = "docs/"
+publish = "public/"
+command = "hugo -b $DEPLOY_PRIME_URL"
+
+[build.environment]
+HUGO_VERSION = "0.105.0"
```

Comment:
you could add a renovate comment here and add toml files to the renovate config so that we get updates for hugo through renovate.
you can find an example for such a comment [here](https://github.com/keptn/lifecycle-toolkit/blob/d7a2b486dd90ff173edbab49ff59988d58cc53c1/.github/workflows/release.yml#L16)
and you need to add toml files to renovate [here](https://github.com/keptn/lifecycle-toolkit/blob/c36673943e7ff54e2921fe6b21ad531603f367aa/renovate.json#L57)